website/src/components/ListPage.vue

```diff
@@ -76,6 +76,16 @@ export default {
           const date = new Date(event.start.year, event.start.month - 1, event.start.date)
           return date < today
         })
+        .sort((first, second) => {
```

Comment:
Since data is already sorted, it is enough to just call `.reverse()` on the array. :)